src/tests/integration/test-data/foundry-dependency-skip

```diff
@@ -0,0 +1,2 @@
+ArithmeticCallTest.test_double_add()
+ArithmeticCallTest.test_double_add_sub_external(uint256,uint256,uint256)
```

Comment:
Let's run these on CI.

Reply:
Or at least, let's run the first one, `test_double_add`, on CI. Then we'll have 2 tests running (`test_double_add` and `test_double_add_double_sub`).

Reply:
There are currently 2 tests running. `test_double_add(uint256,uint256)` and `test_double_add_double_sub(uint256,uint256)`. `ArithmeticCallTest.test_double_add_sub_external(uint256,uint256,uint256)` is currently crashing with `Could not convert ML predicate to sort Bool`.